dpctl-capi/include/dpctl_sycl_program_interface.h

```diff
@@ -85,10 +85,10 @@ DPPLProgram_CreateFromOCLSource (__dppl_keep const DPPLSyclContextRef Ctx,
  * @param    KernelName     Name of kernel
  * @return   A SyclKernel reference if the kernel exists, else NULL
  */
-DPPL_API
-__dppl_give DPPLSyclKernelRef
-DPPLProgram_GetKernel (__dppl_keep DPPLSyclProgramRef PRef,
-                       __dppl_keep const char *KernelName);
+DPCTL_API
+__dpctl_give DPCTLSyclKernelRef
+DPCTLProgram_GetKernel (__dpctl_keep DPCTLSyclProgramRef PRef,
+                       __dpctl_keep const char *KernelName);
```

Comment:
leading white space.

Reply:
Unfortunately we have no C/C++ formatter yet.